tools/adventure-pack/goodies/typescript/Iterator.prototype.find/index.test.ts

```diff
@@ -0,0 +1,58 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "./index";
```

Comment:
The changes from #140 may be relevant here as well.

Reply:
ack!